packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -118,6 +118,14 @@ export async function ejectAsync(projectRoot: string, options?: EjectAsyncOption
     )}`
   );
 
+  if (usesAssetBundlePatterns(projectRoot)) {
+    log.nested(
+      `- 📁 In the bare workflow, ${chalk.yellow(
+        `${'assetBundlePatterns'}`
+      )} no longer affects your builds. To learn about assets in the bare workflow, read here: https://docs.expo.io/bare/updating-your-app/#embedding-assets`
```

Comment:
```suggestion
      `- 📁 The property ${chalk.bold(
        `'assetBundlePatterns'`
      )} is not used in the bare-workflow. ${learnMore('https://docs.expo.io/bare/updating-your-app/#embedding-assets')}`
```

- We use bold instead of color changes for this kind of thing since it works for more terminal styles. 
- Use the `learnMore` link for keeping links as concise as possible.
- "no longer affects" kinda sounds like it's been deprecated.